README.md

```diff
@@ -167,7 +167,7 @@ Compliance Masonry examples in the wild (in order of completeness):
 
 ## Documentation Format
 
-Compliance Masonry uses the [OpenControl v2 Schema](https://github.com/opencontrol/schemas).
+Compliance Masonry uses the [OpenControl Schema](https://github.com/opencontrol/schemas), `v2.0.0` through `v3.1.0`.
```

Comment:
Maybe we want to just put the link and not specify a version. opencontrol.yaml uses only 1.0.0 and then component has it's own range, could get confusing to try to specify here.